tpDonutSimulation.html

```diff
@@ -0,0 +1,40 @@
+
+<script>
+
+
+function Tpdonuts (location, foot_traffic, percent_entering, avg_donuts_ordered) {
+  var hours = 11;
+  this.run = function() {
+    for (var i = 0; i < hours; i++) {
+      var getRandomArbitrary = function(min, max) {
```

Comment:
getRandomArbitrary should be a property (specifically, a method) on the Tpdonuts constructor